media-sound/qpwgraph/qpwgraph-0.2.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
We have EAPI=8 now. While EAPI=7 might have been the up-to-date version back when you submitted the PR, now EAPI=8 is the most recent version, so please bump the EAPI. The transition 7->8 is probably not that much work, since there were not major changes in the EAPI concerning “regular” ebuilds.